src/pages/Trade/components/OrderBookCard/OrderBookCard.tsx

```diff
@@ -35,21 +54,24 @@ const OrderBookCard = ({
 			{
 				name: t('trade.order-book-card.tabs.your-orders'),
 				id: 'yourOrder',
+				// @ts-ignore
 				component: <MyOrders />,
```

Comment:
the ignore needed here because TypeScript complains about `MyOrders` which is in JS (Its actually in TypeScript already in the limit orders PR).